src/openrct2/rct2/S6Importer.cpp

```diff
@@ -841,7 +841,7 @@ class S6Importer final : public IParkImporter
         {
             int32_t quadIndex = rideType >> 5;
             int32_t bitIndex = rideType & 0x1F;
-            bool invented = (_s6.researched_ride_types[quadIndex] & ((uint32_t)1 << bitIndex));
+            bool invented = (_s6.researched_ride_types[quadIndex] & (static_cast<uint32_t>(1) << bitIndex));
```

Comment:
I probably should have been clearer, but I would like you to replace all occurrences of `static_cast<uint32_t>(1)` with `1ULL` in this file (not just the one on line 874).